modules/apigw/outputs.tf

```diff
@@ -6,6 +6,14 @@ output "url" {
   value = aws_api_gateway_deployment.deployment.invoke_url
 }
 
+output "url_hostname" {
```

Comment:
I would prefer to have an output `invoke_url` that is literally just equal to `aws_api_gateway_deployment.deployment.invoke_url`. I understand what you've done here with the regex but it's pretty specific to the purpose you're trying to use it for in the future. Outputting just the raw `invoke_url` might be more reusable.

Reply:
Yeah... I was thinking it would be more convenient if in the future we wanted more headless services with private api gateways... but I can see how it's pretty specific and not really right. Will change, thanks